gc/include/RootScannerTypes.h

```diff
@@ -64,7 +64,8 @@ typedef enum RootScannerEntity {
 	RootScannerEntity_MonitorLookupCachesComplete,
 	RootScannerEntity_MonitorReferenceObjectsComplete,
 	RootScannerEntity_DoubleMappedObjects, /* Obsolete */
-	RootScannerEntity_DoubleMappedOrVirtualLargeObjectHeapObjects,
+	RootScannerEntity_DoubleMappedOrVirtualLargeObjectHeapObjects, /* Obsolete */
```

Comment:
this is probably ok just to replace  rather than obsolete since it's never been referenced by any other code